jplag.endToEndTesting/src/main/java/de/jplag/end_to_end_testing/constants/Constant.java

```diff
@@ -0,0 +1,37 @@
+package de.jplag.end_to_end_testing.constants;
+
+import java.nio.file.Path;
+
+/**
+ * All constant values that are needed in the test cases or helper classes.
+ */
+public final class Constant {
+
+    private Constant() {
+        // private constructor to prevent instantiation
+    }
+
+    /**
+     * Name for the folder to copy the submissions to in order to test them with JPlag
+     */
+    public static final String TEMPORARY_DIRECTORY_NAME = "submission";
+    /**
+     * Identifier to find the system specific temporary directory.
+     */
+    private static final String TEMPORARY_SYSTEM_DIRECTORY = "java.io.tmpdir";
+    /**
+     * Create the complete path to the submission files. Here the temporary system path is extended with the
+     * "TEMPORARY_DIRECTORY_NAME", which is predefined in this class.
+     */
+    public static final String TEMPORARY_SUBMISSION_DIRECTORY_NAME = Path.of(System.getProperty(TEMPORARY_SYSTEM_DIRECTORY), TEMPORARY_DIRECTORY_NAME)
+            .toString();
```

Comment:
You can also use `target/testing-directory-submission` or something similar. Then you do not need to manage cleanups since `target` will be cleared if maven executes clean

Reply:
This would also have been an option but I have to clean the folder after each test run because I use the temporary folder as storage for the current test files. So I definitely need the cleanup after each test.

Reply:
But I think it makes sense to put it in the target folder. As you said, this folder is cleaned anyway, which can prevent errors in the end.